live-examples/js-examples/promise/promise-constructor.html

```diff
@@ -11,7 +11,7 @@
 });
 
 console.log(promise1);
-// expected output: [object Promise]
+// expected output: Promise { <state>: "pending" }
```

Comment:
The value for `expected output` should be the result shown in the interactive example console, not in the browser's JS console. In this case the example displays the array as: `[object Promise]`, so this comment change is not quite accurate.

(Ideally we would have tests for this, but that's quite hard.)

Reply:
I have made the changes as requested. Thank you for the feedback